testing/tekton/examples/README.md

```diff
@@ -0,0 +1,10 @@
+## Tekton Examples
```

Comment:
Can these live in the KIT repo and we point to them?

Reply:
I don't think living in KIT Repo here is a good idea. These are configs specific to Karpenter. These are placeholders to give an idea of structure and will be moved to test suite folders later. In addition, we want to be able to control the files that Tekton will be using to test as well, so we should keep it in our repo.